spatialpandas/io/parquet.py

```diff
@@ -2,29 +2,27 @@
 import json
 import pathlib
 from functools import reduce
+from numbers import Number
+from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
 
+import fsspec
 import pandas as pd
+import pyarrow as pa
 from dask import delayed
-from dask.dataframe import ( # noqa
-    to_parquet as dd_to_parquet,  read_parquet as dd_read_parquet,
-    from_delayed, from_pandas,
-)
+from dask.dataframe import from_delayed, from_pandas
+from dask.dataframe import read_parquet as dd_read_parquet
+from dask.dataframe import to_parquet as dd_to_parquet  # noqa
 from dask.utils import natural_sort_key
-
-from pandas.io.parquet import (
-    to_parquet as pd_to_parquet,
-)
-
-import pyarrow as pa
+from pandas.io.parquet import to_parquet as pd_to_parquet
 from pyarrow import parquet as pq
-from spatialpandas.io.utils import validate_coerce_filesystem
+
 from spatialpandas import GeoDataFrame
 from spatialpandas.dask import DaskGeoDataFrame
+from spatialpandas.geometry import (GeometryDtype, LineDtype, MultiLineDtype,
+                                    MultiPointDtype, MultiPolygonDtype,
+                                    PointDtype, PolygonDtype, RingDtype)
 from spatialpandas.geometry.base import to_geometry_array
-from spatialpandas.geometry import (
-    PointDtype, MultiPointDtype, RingDtype, LineDtype,
-    MultiLineDtype, PolygonDtype, MultiPolygonDtype, GeometryDtype
-)
+from spatialpandas.io.utils import validate_coerce_filesystem
```

Comment:
Shouldn't these be absolute or relative imports, not importing by the name `spatialpandas`, so that there is no ambiguity about them being from this particular package?

Reply:
Thanks!